src/js/game/LevelMVC/LevelPlane.js

```diff
@@ -355,6 +338,20 @@ module.exports = class LevelPlane {
     return posToRefresh;
   }
 
+  positionEquivalence(lhs, rhs) {
```

Comment:
nit: we've already got one of these defined in `GameController`. I vote we either use that or pull it out into a utility method

Reply:
@Hamms I wholeheartedly vote for a utility method. I want to say there are a number of odd helper functions throughout this codebase that could be pulled out into a utility class. I don't know if we care to do that now / later /whenever but I think that's a great scheme to work with.